From 0641b687be37d5520c27897024dfd69df1c398e4 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Wed, 21 Jul 2021 00:55:54 +0530
Subject: [PATCH 11/23] Include missing header for std::isdigit()

This patch resolves the following error with the MSVC compiler:

\\sql\server_component\clone_protocol_service.cc(248):
   error C2039: 'isdigit': is not a member of 'std'
\\path\to\include\cstdarg(17): note: see declaration of 'std'
---
 sql/server_component/clone_protocol_service.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sql/server_component/clone_protocol_service.cc b/sql/server_component/clone_protocol_service.cc
index 1b626b072eb..7287ae08f7c 100644
--- a/sql/server_component/clone_protocol_service.cc
+++ b/sql/server_component/clone_protocol_service.cc
@@ -42,6 +42,8 @@
 #include "sql/dd/cache/dictionary_client.h"
 #include "sql/dd/dictionary.h"
 
+#include <cctype>
+
 struct CHARSET_INFO;
 
 /** The minimum idle timeout in seconds. It is kept at 8 hours which is also
-- 
2.48.1

